src/api.ts

```diff
@@ -0,0 +1,185 @@
+const base = 'http://localhost:3000';
+
+const garage = `${base}/garage`;
+const engine = `${base}/engine`;
+const winners = `${base}/winners`;
+
+export const getCars = async (
+    page: number,
+    limit = 7
+): Promise<{
+    items: { id: number; color: string; name: string }[];
```

Comment:
The absense of types or interfaces makes code hard to read. I.e. we could specify the Car interface like {id: number; color: string; name; string}, so the code would look like Promise<{items: Car[], count: string}>.

We can further add an interface for this like CarResponse {items: Car[], count: string}, so the Promise would look like: Promise<CarResponse>